frontend/src/components/InputTodo.js

```diff
@@ -2,7 +2,7 @@ import { Fragment, useState } from "react";
 
 const InputTodo = () => {
 
-    const [description, setDescription] = useState('hello')
+    const [description, setDescription] = useState('default')
```

Comment:
creates a todo with that description even when input is empty, shouldnt be like this